morceus-data/latin/stems/nominals/ls.nom

```diff
@@ -136272,6 +136360,7 @@ nu_gu^lae
 Nuitho_nes		
 :le:Nuithones
 :no:Nuitho_n	0_is masc pl
+:no:Nuito_n	0_is masc pl
```

Comment:
variant?

Reply:
added orth tag (I think the appropriate one for h's appearing or disappearing in foreign place names)